cypress/e2e/login_management/invalidCredentials.feature

```diff
@@ -3,8 +3,9 @@ Feature: Login Management - Verify login functionality via wrong user data
 
```

Comment:
since this test is still unstable so I would keep it in the previous state, we have a follow up ticket for it.

Reply:
done!